yaml/_loader_state.ts

```diff
@@ -210,1069 +210,898 @@ export class LoaderState {
     this.onWarning?.(error);
   }
 
-  readDocument() {
```

Comment:
Why was `readDocument()` moved?

Reply:
It wasn't, `readDocument()` and `readDocuments()` were at the end of the class where they still are.